linetools/analysis/absline.py

```diff
@@ -284,3 +290,155 @@ def sum_logN(obj1, obj2):
             pass # Take sums
     # Return
     return flag_N, logN, sig_logN
+
+
+def get_tau0(wrest, fosc, N, b):
+    """Get the value of the optical depth at the line center,
+    tau0. Taken from Draine 2011 (see Chapter 9). It neglects stimulated
+    emission which is fine for IGM or ISM except for radio-frequency
+    transitions.
+
+    Parameters
+    ----------
+    wrest : Quantity
+        Rest-frame wavelength of the transition
+    fosc : float
+        Oscillator strength of the transition
+    N : Quantity or Quantity array
+        Column density
+    b : Quantity or Quantity array of same shape as logN
+        Doppler parameter
+
+    Returns
+    -------
+    tau0: float or array
+        Optical depth at the line center. If logN and b are
+        arrays they must be of same shape.
+    """
+    # check format for N and b
+    if isiterable(N):
+        if np.shape(N) != np.shape(b):
+            raise IOError('If N is array, b must be array of same shape.')
+
+    # convert to CGS
+    b_cgs = b.to('cm/s')
```

Comment:
I'd like to leave these lines though because they help to check for the correct units for the input parameters, and will raise meaningful errors for the user if the units are wrong.